task2_Gal_Malka/include/IMovement.h

```diff
@@ -0,0 +1,98 @@
+#pragma once
+
+using Direction = std::pair<int, int>;
+
+// Regular directions
+const Direction up = { -1, 0 };
```

Comment:
Nice, I liked these Direction consts. It's always a good idea to give things names. Makes codes more readable